src/core/NewDomain.ml

```diff
@@ -900,8 +900,13 @@ struct
 
   let occurs xs =
```

Comment:
oh lol, you were right! I forgot that I had written something like this :laughing: